build.gradle

```diff
@@ -115,41 +110,22 @@ subprojects {Project subproject ->
     dependencies {
         documentation "org.codehaus.groovy:groovy-templates:$groovyVersion"
         documentation "org.codehaus.groovy:groovy-dateutil:$groovyVersion"
-
-        testCompile "cglib:cglib-nodep:3.3.0"
-        testCompile "org.objenesis:objenesis:3.1"
-        testCompile("org.spockframework:spock-core:${spockVersion}") {
+        testImplementation "cglib:cglib-nodep:3.3.0"
+        testImplementation "org.objenesis:objenesis:3.3"
+        testImplementation("org.spockframework:spock-core:${spockVersion}") {
             exclude module:'groovy-all'
         }
-        testCompile "io.micronaut.test:micronaut-test-spock:$micronautTestVersion"
-        testCompile "org.codehaus.groovy:groovy-test:$groovyVersion"
-
-        testRuntime "ch.qos.logback:logback-classic:1.2.3"
+        testImplementation "io.micronaut.test:micronaut-test-spock:$micronautTestVersion"
+        testImplementation "org.codehaus.groovy:groovy-test:$groovyVersion"
+        testRuntimeOnly "ch.qos.logback:logback-classic:1.4.1"
     }
 
     groovydoc.classpath += project.configurations.documentation
 
     task allDeps(type: DependencyReportTask) {}
 
-    apply plugin: 'checkstyle'
-
-    checkstyle {
-        toolVersion = 8.10
-        configFile = rootProject.file('config/checkstyle/checkstyle.xml')
-
-        // Per submodule
-        maxErrors = 1
-        maxWarnings = 10
-
-        showViolations = true
-    }
-
-    checkstyleTest.enabled = false
-
```

Comment:
why does this remove checkstyle?

Reply:
It's unclear the usage and local build fails with that.

The XML was introduced but sounds to not be tested.
https://github.com/couchbaselabs/couchbase-micronaut/blob/main/config/checkstyle/checkstyle.xml 

Without any change regarding the current PR, runned in clean branch from master, the issue on checkstyle also popups in the pipeline.
https://github.com/couchbaselabs/couchbase-micronaut/actions/runs/3260252338/jobs/5353681285